pkg/kubelet/apis/kubeletconfig/v1alpha1/types.go

```diff
@@ -197,6 +197,8 @@ type KubeletConfiguration struct {
 	// The CIDR to use for pod IP addresses, only used in standalone mode.
 	// In cluster mode, this is obtained from the master.
 	PodCIDR string `json:"podCIDR"`
+	// PodPidsLimit is the maximum number of pids in any pod.
+	PodPidsLimit int64
```

Comment:
I think this should be a pointer, and also it needs a json tag.

Reply:
done